plugins/sinks/compass/sink.go

```diff
@@ -292,7 +293,9 @@ func (*Sink) getLabelValueFromProperties(field1, field2 string, asset *v1beta2.A
 
 func init() {
 	if err := registry.Sinks.Register("compass", func() plugins.Syncer {
-		return New(&http.Client{}, plugins.GetLog())
+		return New(&http.Client{
+			Transport: telemetry.NewHTTPTransport(nil),
+		}, plugins.GetLog())
```

Comment:
Can we change the constructor instead so we are not dependent on caller for instrumentation? Won't be very elegant but we can do it with a type assertion check

Reply:
applies for http and stencil sinks as well